### PR TITLE
Gherkin and automation for helm chart registry to be namespace scoped

### PR DESCRIPTION
**Pull request by sanketpathak**
Epic: https://issues.redhat.com/browse/ODC-5713
Story: https://issues.redhat.com/browse/ODC-6419
Acceptance criteria:

- As a non privileged user, I should be able to connect my Helm Chart Repository to my project, so I can access the Helm Charts from the DevConsole catalog

**Checks required for approving Epic gherkin scripts PR:**

- [x] Add @epic-number to the scenarios or feature file [e.g: @odc-xxx]
- [x] Add @to-do for automation possible scenarios
- [x] Add @regression or @smoke based on the importance of the scenario
- [x] Update the test scenarios count in [Automation status confluence Report](https://docs.jboss.org/display/ODC/Automation+Status+Report)
- [x] Check for the linter issues by executing `yarn run gherkin-lint` on frontend folder [Skip epic number tags related linter issues]

**Fix**: https://issues.redhat.com/browse/ODC-6470

**Test Setup**:

Cluster should be running on local

Check the TAGS in frontend/packages/topology/integration-tests/cypress.json file be: ""(@pre-condition or @smoke or @regression) and @[odc-5713](https://issues.redhat.com/browse/odc-5713) and not (@manual or @to-do or @un-verified or @broken-test)"",

Command to execute:

Run a cluster locally and execute the command `npm run test-cypress-helm` simultaneously

Execute the install-helm-chart.feature file

**Browser**
Chrome 93

**Test Execution Screenshot:**

### CODE REVIEW

**Comment by jrichter1** [comment]
Run is green
/lgtm

File: frontend/packages/helm-plugin/integration-tests/features/helm/install-helm-chart.feature [hunk 1]
```diff
@@ -117,3 +119,30 @@ Feature: Install the Helm Release
...
               And user clicks on helm chart with blue tick
              Then user will see Blue certified badge associated with heading of the helm chart
+
+
+        @regression @odc-5713
+        Scenario Outline: Namespace-scoped Helm Chart Repositories in the dev catalog: HR-06-TC12
+            Given user is at Add page
+            # Uncomment below for cluster not having projecthelmchartrepositories CRD
+            #   And user has applied namespaced CRD yaml "<crd_yaml>"
+              And user has created namespaced helm chart repo with yaml "<cr_yaml>" in namespace "aut-helm"
```
Comment (debsmita1) [inline frontend/packages/helm-plugin/integration-tests/features/helm/install-helm-chart.feature:129]:
```suggestion
              And user has added namespaced helm chart repo with yaml "<cr_yaml>" in namespace "aut-helm"
```

**Comment by debsmita1** [comment]
you need to add the scenario where the user is able to see the quick starts link in the helm dev catalog description (see jira ticket https://issues.redhat.com/browse/ODC-6466)

File: frontend/packages/helm-plugin/integration-tests/features/helm/install-helm-chart.feature (hunk 1 above)
Comment (sanketpathak) [reply frontend/packages/helm-plugin/integration-tests/features/helm/install-helm-chart.feature]: Updated

**Comment by debsmita1** [comment]
Thanks @sanketpathak ! 
/lgtm

**Comment by sanketpathak** [comment]
This PR only affects our e2e tests.
/label qe-approved
/label px-approved
/label docs-approved

**Pull request merged**